assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -208,6 +285,19 @@ export const selectors = {
 
 		return error || null;
 	},
+
+	/**
+	 * Checks whether accounts are currently being fetched.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean} Whether accounts are currently being fetched or not.
+	 */
+	isFetchingAccounts( state ) {
```

Comment:
@felixarntz should this be `isDoingGetAccounts` instead?

Reply:
If this is intended for public usage, yes. `createFetchStore` will be adding selectors like `isFetching...`, but those are private, for testing etc. If we need a public one for e.g. usage by components we should add an `isDoing...` one (with `createFetchStore` that would typically just be calling the `isFetch...` one).

Reply:
Okay, yeah - not sure if it needs to be public (it's marked `@private`) as it is so, I guess that's fine for now 👍